### PR TITLE
 Feature/style clean up + Add logic for hero image                            

### PR DESCRIPTION
**Pull request by auredufour**
Change the scss for consistency:
- create a constants file with variables and mixins
- create a general style file
- clean-up the css.

### CODE REVIEW

**Comment by srhboo** [comment]
👍 looks cool!

File: src/components/Home.js [hunk 1]
```diff
@@ -3,9 +3,22 @@ import { Button } from './Button';
...
+const randomNumber = (max, min) => {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+let classNumber;
+
+const headerClass = () => {
+  if (!classNumber){
+    classNumber = randomNumber(0 , 7);
+  }
```
Comment (cgolinski) [inline src/components/Home.js]: The random images seem to be working, but it looks like this is backwards. Shouldn't be it randomNumber(7, 0)?

**Comment by cgolinski** [comment]
Didn't have a chance to thoroughly review code, but the app looks good. One exception I found: the button within the set location modal seems too small. Positioning in that modal might also need to be updated.

**Comment by auredufour** [comment]
@cgolinski I updated the button, I created a general class that I forgot to add to the buttons! What do you mean by positioning?

**Comment by cgolinski** [comment]
Sorry, was rushed when I wrote that. The button wasn't showing until you
after you have selected an address. Wasn't sure if that was intentional, or
if the modal size needs to be adjusted.

File: src/components/Home.js (hunk 1 above)
Comment (auredufour) [inline src/components/Home.js]: Ahah! Yes! It should be the other way!

Comment (auredufour) [inline src/components/Home.js]: Ahah! Yes! It should be the other way!

**Pull request merged**